widgetssdk/src/main/java/com/glia/widgets/core/configuration/GliaSdkConfiguration.java

```diff
@@ -5,7 +5,7 @@
 import com.glia.androidsdk.screensharing.ScreenSharing;
 import com.glia.widgets.GliaWidgets;
 import com.glia.widgets.UiTheme;
-import com.glia.widgets.view.unifieduiconfig.component.RemoteConfiguration;
+import com.glia.widgets.view.unifiedui.config.RemoteConfiguration;
```

Comment:
I think according to code convention we need to name it like this: `unified_ui`, don't we?

Reply:
according to the official [documentation/convention](https://developer.android.com/kotlin/style-guide#package_names), this is not allowed.

Reply:
Sorry, yes, you are right.